frontend/src/components/Header/NavTop.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
```

Comment:
It looks like you've removed all the admin logic; we're going to need that, so please add it back in here.